biolink-model.yaml

```diff
@@ -194,6 +193,10 @@ subsets:
     description: >-
       Sample/biosample datamodel
 
+  ecolink:
```

Comment:
now that we have a ecolink linkml schema, maybe we can change the subsets to mappings? :)